ddtrace/opentelemetry/tracer_test.go

```diff
@@ -85,6 +89,74 @@ func TestTracerOptions(t *testing.T) {
 	assert.Contains(fmt.Sprint(sp), "dd.env=wrapper_env")
 }
 
+func TestSpanContext(t *testing.T) {
+	assert := assert.New(t)
+	tp := NewTracerProvider()
+	defer tp.Shutdown()
+	otel.SetTracerProvider(tp)
+	tr := otel.Tracer("")
+
+	pCtx, parent := tr.Start(context.Background(), "parent")
+	pSpanCtx := parent.SpanContext()
+
+	_, child := tr.Start(pCtx, "child")
+	cSpanCtx := child.SpanContext()
+
+	assert.Equal(cSpanCtx.TraceFlags(), pSpanCtx.TraceFlags())
+	assert.Equal(cSpanCtx.TraceID(), pSpanCtx.TraceID())
+	assert.Equal(cSpanCtx.IsRemote(), pSpanCtx.IsRemote())
+	assert.Equal(cSpanCtx.TraceState().String(), pSpanCtx.TraceState().String())
+}
+
+func TestForceFlush(t *testing.T) {
+	assert := assert.New(t)
+	testData := []struct {
+		timeOut time.Duration
+		flushed bool
+	}{
+		{timeOut: 0 * time.Second, flushed: false},
+		{timeOut: 300 * time.Second, flushed: true},
+	}
+	success := false
+	setFlushFail := func(ok bool) { success = ok }
+
+	for _, tc := range testData {
```

Comment:
When you're doing a table driven test, you typically don't want to just loop over the test cases, but you also want to call `t.Run` to run a subtest. Check out some examples of this throughout our tests in the tracer package
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests